universal-login-web3/README.md

```diff
@@ -1,4 +1,7 @@
-# Using Web3 providers
+# Web3
+Web3 is deprecated. Using it is not recommended.
```

Comment:
Add something like `use @unilogin/provider instead` and you can add link to README or npm